src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ChangeApprenticeship.feature

```diff
@@ -64,3 +65,10 @@ Scenario: Notify user of change
 	And we have an update apprenticeship request
 	When the update is posted
 	Then send a Change of Circumstance email to the user
+
+Scenario: Trying to update apprenticeship which doesn't have an email
+	Given we do not have an existing apprenticeship, confirmed or unconfirmed
+	And we have an update apprenticeship request without an email
+	When the update is posted
+	Then the response is bad request
+	And a validation exception is thrown for the field: "Email" 
```

Comment:
I think that a validation exception will result in either a HTTP status of 400 or 500 returned to the outer API, and in turn to the Jobs project.

This will cause the `ApprenticeshipUpdatedApprovedEvent` to be marked as errored, and put back in the queue to retry.  No matter how many times it is retried, the lack of email address won't change.

I wonder if it makes sense to log this as an error, but return a successful HTTP status?  This would avoid unnecessary retries that will always fail.

Reply:
Never mind - I've just seen it's taken care of in the APIM layer